release-binaries.nix

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+let
+  protocols = import ./protocols.nix;
+  protocolsFormatted = builtins.concatStringsSep ", " (protocols.allowed ++ protocols.active);
+in[
+  {
+    name = "tezos-client";
```

Comment:
now it also has `tezos-admin-client` binary, which is weird

Reply:
Well, it's built as part of `tezos-client` opam package

Reply:
I see, but still, I'm not sure if it's a good idea to break existing package per binary principle

Reply:
Do I just `rm tezos-admin-client` in `postInstall` or do I create a separate package for it?

Reply:
Let's have a separate package for it as it was done previously

Reply:
@rvem done